README.md

```diff
@@ -102,3 +102,244 @@ This will start the server with auto-reload.
 ```bash
 curl http://<hostname>:<port>/healthcheck
 ```
+
+## API
+
+```bash
+POST /data
+```
+
+Stores the device message received from the webhook service.
+
+### Parameters
+#### Head Parameters
+
+Field | Required | Description
+--- | --- | ---
+auth_token | Y | User's authentication token
+
+### Body Parameters
+
+Field | Required | Description
+--- | --- | ---
+data | Y | The message fata sent by the device. This data includes the device that it was sent and the payload.
+
+### Example
+#### Request
+
+```bash
+POST https://storage.knot.cloud/data
+```
+
+##### Request Body
+```json
+{
+	"from": "188824f0-28c4-475b-ab36-2505402bebcb",
+	"payload": {
+	    "sensor_id": 2,
+    	"value": 234
+	}
+}
+```
+
+##### Response
+```json
```

Comment:
We don't return this response anymore, just consider `201 - created` as [here](https://github.com/CESARBR/knot-cloud-storage#response).

Reply:
Done!